components/Layout/HomeHeader/HomeHeaderItems.vue

```diff
@@ -174,6 +190,34 @@ export default Vue.extend({
   },
   methods: {
     ...mapActions('playbackManager', ['play']),
+    getItemIcon(item: BaseItemDto): string {
```

Comment:
use `getLibraryIcon` in itemsUtils?

Reply:
They don't do the same thing. One is for libraries, this one for items.

We can probably make a mixin for it at a later date, but I'd rather not mess with more components than already do here.